servers/visual/shader_language.cpp

```diff
@@ -2369,6 +2369,21 @@ Variant ShaderLanguage::constant_value_to_variant(const Vector<ShaderLanguage::C
 				value = Variant(t);
 				break;
 			}
+			case ShaderLanguage::TYPE_ISAMPLER2DARRAY:
+			case ShaderLanguage::TYPE_ISAMPLER2D:
+			case ShaderLanguage::TYPE_ISAMPLER3D:
+			case ShaderLanguage::TYPE_SAMPLER2DARRAY:
+			case ShaderLanguage::TYPE_SAMPLER2D:
+			case ShaderLanguage::TYPE_SAMPLER3D:
+			case ShaderLanguage::TYPE_SAMPLERCUBE:
+			case ShaderLanguage::TYPE_USAMPLER2DARRAY:
+			case ShaderLanguage::TYPE_USAMPLER2D:
+			case ShaderLanguage::TYPE_USAMPLER3D: {
+				// FIXME: Implement?
+				break;
+			}
+			case ShaderLanguage::TYPE_VOID:
+				break;
```

Comment:
```
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_ISAMPLER2DARRAY' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_ISAMPLER2D' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_ISAMPLER3D' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_SAMPLER2DARRAY' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_SAMPLER2D' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_SAMPLER3D' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_SAMPLERCUBE' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_USAMPLER2DARRAY' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_USAMPLER2D' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_USAMPLER3D' not handled in switch [-Wswitch]
servers/visual/shader_language.cpp:2297:10: warning: enumeration value 'TYPE_VOID' not handled in switch [-Wswitch]
```
@karroffel @reduz

Reply:
Those are texture types so I guess no relevant in `constant_value_to_variant`, I'm adding a comment to make it clear.